### PR TITLE
Fix unescaped arguments in websockify command

### PR DESCRIPTION
**Pull request by tutacat**
When passing a path with spaces to websockify, it is incorrectly read as multiple arguments, because the args weren't enclosed in quotes.

### CODE REVIEW

File: utils/novnc_proxy [hunk 1]
```diff
@@ -93,14 +93,14 @@ while [ "$*" ]; do
     --web)     WEB="${OPTARG}"; shift            ;;
     --ssl-only) SSLONLY="--ssl-only"             ;;
     --file-only) FILEONLY_ARG="--file-only"      ;;
-    --record) RECORD_ARG="--record ${OPTARG}"; shift ;;
-    --syslog) SYSLOG_ARG="--syslog ${OPTARG}"; shift ;;
+    --record) RECORD_ARG='--record "${OPTARG}"'; shift ;;
+    --syslog) SYSLOG_ARG='--syslog "${OPTARG}"'; shift ;;
```
Comment (CendioOssman) [inline utils/novnc_proxy:97]: I'm afraid `'` disables variable expansion, so this won't work.

**Review by CendioOssman — CHANGES_REQUESTED** [review]
Thank you for your fix!

I'm afraid it isn't entirely correct, though. Please have a look at the comment.

**Comment by samhed** [comment]
@tutacat do you think you will get a chance to fix this?

**Comment by CendioOssman** [comment]
This looks better, but unfortunately, it still doesn't work. It is still splitting the words for the record, syslog and auth arguments.

**Comment by CendioOssman** [comment]
No response. Closing.

**Comment by tutacat** [comment]
Yes. I am a "volounteer". Not 100% of my time is github. Not all minds work the same, either.

**Comment by CendioOssman** [comment]
I understand, and we do try to give some time before we consider something abanonded.

I can gladly reopen it if it's not abandoned and you'd like to resume the work?

**Comment by tutacat** [comment]
hiya sorry about this pull, it was easier to make [a new one](1891)

**Pull request closed without merging**